prow/cmd/deck/jobs.go

```diff
@@ -114,11 +116,22 @@ func (ja *JobAgent) GetJobLog(job, id string) ([]byte, error) {
 	if j.Spec.Agent == kube.KubernetesAgent {
 		return ja.pkc.GetLog(j.Status.PodName)
 	}
-	num, err := strconv.Atoi(id)
-	if err != nil {
-		return nil, err
+	for _, agentToTmpl := range ja.c.Config().Deck.ExternalAgentLogs {
+		if agentToTmpl.Agent != string(j.Spec.Agent) {
+			continue
+		}
+		var b bytes.Buffer
+		if err := agentToTmpl.URLTemplate.Execute(&b, &j); err != nil {
+			return nil, fmt.Errorf("error executing URL template: %v", err)
```

Comment:
Include the agent and prowjob in the error message like you do bellow.

Reply:
done